WORKSPACE

```diff
@@ -2,12 +2,15 @@ workspace(name = "com_github_grpc_grpc")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("//bazel:grpc_deps.bzl", "grpc_deps", "grpc_test_only_deps")
+load("//bazel:grpc_python_deps.bzl", "grpc_python_deps")
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 grpc_deps()
 
 grpc_test_only_deps()
 
+grpc_python_deps()
```

Comment:
This should be redundant. [It's already called by `grpc_deps()`.](https://github.com/grpc/grpc/blob/690e313c3886eb1e766a6dce2ecb9fe6db4591ac/bazel/grpc_deps.bzl#L233)

Reply:
Sorry, done